lib/include/openamp/rpmsg.h

```diff
@@ -506,29 +508,39 @@ static inline int rpmsg_send_nocopy(struct rpmsg_endpoint *ept,
  *
  * As an option Some rpmsg clients can specify an endpoint with a specific
  * source address.
+ *
+ * @param ept		Pointer to rpmsg endpoint
+ * @param rdev		RPMsg device associated with the endpoint
+ * @param name		Service name associated to the endpoint
+ * @param src		Local address of the endpoint
+ * @param dest		Target address of the endpoint
+ * @param cb		Endpoint callback
+ * @param ns_unbind_cb	Endpoint service unbind callback, called when remote
+ *			ept is destroyed.
+ *
+ * @return 0 on success, or negative error value on failure.
  */
 
```

Comment:
could you take the opportunity to remove the extra line

Reply:
Hi @arnopo Do you mean line 520? This is following the established convention of a blank line following the parameter descriptions before the return description.  I think it looks like there is an extra blank line because the description for ns_unbind_cb consumes two lines. Please let me know if you still want 520 removed.

Reply:
Hi @tammyleino, the blank line to remove is on the line 523